File: pubnub-gson/src/main/java/com/pubnub/api/endpoints/channel_groups/AddChannelChannelGroup.java
package com.pubnub.api.endpoints.channel_groups;

import com.pubnub.api.endpoints.Endpoint;
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsAddChannelResult;

public interface AddChannelChannelGroup extends Endpoint<PNChannelGroupsAddChannelResult> {
    AddChannelChannelGroup channelGroup(String channelGroup);

    AddChannelChannelGroup channels(java.util.List<String> channels);
}

File: pubnub-kotlin/pubnub-kotlin-api/src/jvmMain/kotlin/com/pubnub/api/endpoints/channel_groups/AddChannelChannelGroup.kt
package com.pubnub.api.endpoints.channel_groups

import com.pubnub.api.Endpoint
import com.pubnub.api.models.consumer.channel_group.PNChannelGroupsAddChannelResult

/**
 * @see [PubNub.addChannelsToChannelGroup]
 */
interface AddChannelChannelGroup : com.pubnub.kmp.endpoints.channel_groups.AddChannelChannelGroup, Endpoint<PNChannelGroupsAddChannelResult> {
    val channelGroup: String
    val channels: List<String>
}
